apps/ensindexer/src/config/config.schema.ts

```diff
@@ -153,6 +155,10 @@ const DatabaseUrlSchema = z.union(
   },
 );
 
+const RealtimeIndexingGapThresholdSchema = positiveIntegerSchema(
```

Comment:
honestly i don't think this should be configurable in the app, this should just be a constant, and consumers can provide their own gap via the api if necessary. @lightwalker-eth thoughts?

Reply:
@BanaSeba the `/ready` endpoint is reserved by Ponder framework. We cannot override its logic.